src/AzureIoTHub.Portal/Server/Controllers/DevicesController.cs

```diff
@@ -83,11 +83,6 @@ public async Task<IActionResult> CreateDeviceAsync(DeviceDetails device)
         {
             try
             {
-                if (!Eui.TryParse(device.DeviceID, out ulong deviceIdConvert) && device.DeviceType == "LoRa Device")
-                {
-                    throw new InvalidOperationException("the device id is in the wrong format.");
-                }
-
```

Comment:
Why this has been removed ?

Reply:
I didn't want to change this code, but I just corrected it.